### DISCUSSION TITLE
Firefox only: Cameras on main page of 0.14 are often blank

### DISCUSSION THREAD

**Discussion by mintjberry** [blakeblackshear/frigate#13576] [General]
As per title, the individual camera feed on the main frigate UI page in 0.14 is often blank. This wasn't a problem on 0.13.
See image below:

Firefox (v130.0) and Floop (Firefox fork)
![image](https://github.com/user-attachments/assets/3bcb06fd-a66f-4e24-86b0-4003234f3d8e)

On Chrome this is not an issue:
![image](https://github.com/user-attachments/assets/834f4916-2d34-40e8-9c9d-91d62ce5e124)

Am not sure if it's a Frigate issue or Firefox, but would be good to get it fixed or know how to fix it.

**Comment by hawkeye217** [comment]
Please open a support discussion with the required information so we can help.

**Reply by seler** [reply]
I've got the same problem. Some cameras do not work on firefox, but all work on chrome. How can I join the support discussion?

**Reply by mintjberry** [reply]
See here: https://github.com/blakeblackshear/frigate/discussions/13577#discussioncomment-10626050
Specifically the answer, as a workaround you can force the video to be transcoded to H264.
